src/main/java/gregtech/api/gui/widgets/TextFieldWidget.java

```diff
@@ -64,7 +81,10 @@ protected void onSizeUpdate() {
     @Override
     public void drawInBackground(int mouseX, int mouseY, IRenderContext context) {
         super.drawInBackground(mouseX, mouseY, context);
+        FontRenderer fontRenderer = Minecraft.getMinecraft().fontRenderer;
+        fontRenderer.FONT_HEIGHT = fontSize;
         this.textField.drawTextBox();
+        fontRenderer.FONT_HEIGHT = 9;
```

Comment:
iirc, modifying `FONT_HEIGHT` has no effect on the Fontrenderer result, (i mean **size**). It seems that the font size of the vailla Fontrender can only be changed using the scale method. `FONT_HEIGHT` Is used to help Fontrenderer know the default height of different fonts to determine position to render, but not size. please ignore this if I'm wrong, but yd better not touch it.